app/views/calculation/income/currentIncome.scala.html

```diff
@@ -49,7 +49,7 @@
         @formInputMoney(
             field = currentIncomeForm("amount"),
             question = title,
-            hintText = Some(Messages("calc.resident.shares.currentIncome.helpText")),
+            hintText = Some(messages("calc.currencyHint", "£10,000.50")),
```

Comment:
this has overwritten the original help text, the currency example should be appended to that message instead of overwriting it with a generic currency hint

Reply:
fixed on [eabcf42](https://github.com/hmrc/cgt-calculator-resident-shares-frontend/pull/184/commits/eabcf42313323762c7b1140a03fb3c5686aef059)